### PR TITLE
resolves #3994 fix instructions for Confluence XHTML migration

### LINKED ISSUE #3994: Wrong instructions in "Migrate from Confluence XHTML" doc page
The page published here: https://docs.asciidoctor.org/asciidoctor/latest/migrate/confluence-xhtml/ (Retrieved April 7th, 2021 at 12:48PM Spain Mainland Time) describes how to execute a groovy script to acomplish the migration of Confluence HTML.
I checked that and those instructions are wrong, they don't work. I managed to obtain the right result but with modifications to that instructions.

I will attach a PR fixing: https://github.com/asciidoctor/asciidoctor/blob/master/docs/modules/migrate/pages/confluence-xhtml.adoc with the steps that I checked to work.

Any comment will be welcomed.

### PR DESCRIPTION
**Pull request by juliojgd**
Fixes #3994 by modifying the instructions to match actual behavior of the provided groovy script.
Previous instructions were describing a behavior inconsistent with the groovy script provided. 
I've chosen to not change at all the script, only adapt the intructions for them to work.

### CODE REVIEW

File: docs/modules/migrate/pages/confluence-xhtml.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ The script is designed to be run locally on HTML files or directories containing
 
 . Save the script contents to a `convert.groovy` file in a working directory.
 . Make the file executable according to your specific OS requirements.
-. Place individual files, or a directory containing files into the working directory.
-. Run `groovy convert filename.html` to convert a single file.
-. Confirm the output file meets requirements
-. Recurse through a directory by using this command pattern: `groovy convert directory/*.html`
+. Create a `html` directory for input files and a `asciidoc` directory for output files, both inside the working directory.
```
Comment (mojavelinux) [inline docs/modules/migrate/pages/confluence-xhtml.adoc:26]:
```suggestion
. Create an `html` directory for input files and an `asciidoc` directory for output files, both inside the working directory.
```

File: docs/modules/migrate/pages/confluence-xhtml.adoc [hunk 2]
```diff
@@ -23,7 +23,7 @@ The script is designed to be run locally on HTML files or directories containing
...
+. Place individual files, or a directory containing files into the aforementioned `html` directory.
```
Comment (mojavelinux) [inline docs/modules/migrate/pages/confluence-xhtml.adoc:27]:
```suggestion
. Place individual files, or a directory containing files, into the aforementioned `html` directory.
```

File: docs/modules/migrate/pages/confluence-xhtml.adoc [hunk 3]
```diff
@@ -23,7 +23,7 @@ The script is designed to be run locally on HTML files or directories containing
...
+. Run `groovy convert` to convert the files contained inside the `html` directory.
+. Confirm the output file located in the `asciidoc` directory meets requirements.
```
Comment (mojavelinux) [inline docs/modules/migrate/pages/confluence-xhtml.adoc:29]:
```suggestion
. Look for the generated output file in the `asciidoc` directory and confirm it meets your requirements.
```

**Review by mojavelinux — CHANGES_REQUESTED** [review]

File: docs/modules/migrate/pages/confluence-xhtml.adoc (hunk 1 above)
Comment (juliojgd) [reply docs/modules/migrate/pages/confluence-xhtml.adoc:26]: I was doubtful about "a 'html'" vs. "an 'html'" Thanks for clarifying!

**Comment by juliojgd** [comment]
Thank you very much for reading my issue and your review. I saw OK all your proposed improvements :)

**Comment by mojavelinux** [comment]
Super! Thanks for updating!

**Pull request merged**